.circleci/config.yml

```diff
@@ -113,6 +113,48 @@ executors:
           WANDB_ENABLE_TEST_CONTAINER: true
     resource_class: xlarge
 
+  local-testcontainer-importers:
+    parameters:
+      python_version_major: { type: integer }
+      python_version_minor: { type: integer }
+      server_name2:
+        type: string
+        default: localhost-wandb-2
+      local_base_port2:
+        type: string
+        default: "8080"
+      services_api_port2:
+        type: string
+        default: "8083"
+      fixture_service_port2:
+        type: string
+        default: "9015"
+    docker:
+      - image: "python:<<parameters.python_version_major>>.<<parameters.python_version_minor>>"
+      # the src server
+      - image: << pipeline.parameters.container_registry >>/<< pipeline.parameters.gcp_project >>/images/local-testcontainer:<< pipeline.parameters.wandb_server_tag >>
+        auth:
+          username: _json_key
+          password: $GCP_SERVICE_ACCOUNT_JSON_DECODED
+        environment:
+          CI: 1
+          WANDB_ENABLE_TEST_CONTAINER: true
```

Comment:
this is common to the other executor as well... maybe we can take it out and make it it's own executor...

Reply:
I don't understand -- this is already its own executor `local-testcontainer-importers`?

Reply:
sorry, i meant if we could share the local test container across all the executors (that use it... that would be nice...)

Reply:
although looking at the circleci documentation, don't know if there is an easy way (or a way) to do it.

Reply:
Yeah doesn't seem like you can conditionally add the 2nd container